StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -814,6 +843,27 @@ private func CreatePaneViewController(
             assertionFailure("Code logic error. Missing parameters for \(pane).")
             viewController = nil
         }
+    case .networkingSaveToLinkVerification:
+        if
+            let consumerSession = dataManager.consumerSession,
+            let selectedAccountIds = dataManager.linkedAccounts?.map({ $0.id })
```

Comment:
should we also add a condition of selectedAccountIds not being empty?

Reply:
hmm great point! thank you

Reply:
I will change it to actually just pass a single account; that's actually the case

I was debating this the whole time (should I pass the array or single account), and if I had just done the simple thing, this wouldn't have been an issue (I was being fancy)

Reply:
simplicity wins again